privacyidea/static/components/token/views/token.enroll.indexedsecret.html

```diff
@@ -0,0 +1,22 @@
+<p class="help-block" translate>
+    The indexed secret Token is based on a shared secret between privacyIDEA
+    and the user.
+    During authentication the user is asked for random positions from this known
+    secret.
+</p>
+
+<h4 translate>Token data</h4>
+
+<div class="form-group">
+    <label for="description" translate>Secret</label>
+    <input type="text" class="form-control"
+            placeholder="{{ 'Some nice words...'|translate }}"
```

Comment:
Maybe we should disable/hide the input in case a `force_attribute` policy is set.
And what happens if the attribute does not exist/is not available?

Reply:
This would be cool, but a bit more complex, since we also need to pass the force-information to the UI. Will take some time.

Reply:
@plettich I am not sure about this. I looked into this and this is actually not that simple.

For admins: Usually when the admin logs in it gets it UI settings. But this policy can be used dependent. So we would have to fetch the policy anew each time a user is change :-/

For users: Users can not fetch policies. So for users the server would have to pass this information at login time.
This produces a lot of code for a tiny convenience...

Reply:
Ah, oh. Done this for the normal user in 25e65141.
Will also do it for the admin.

Reply:
As discussed, we do not implement the field hiding for admins, since this could end up in contradicting policies, if the ui needs to fetch the policies, but the admin is not allowed to fetch polices.